src/components/Sign-In/Sign-In.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event'
+import '@testing-library/jest-dom';
+
+import SignIn from './Sign-In';
+
+describe('Sign-In', () => {
+  it("Should render the component", () => {
+    const mockSignIn = jest.fn();
+    render(<SignIn logIn={mockSignIn} />)
+    expect(screen.getByRole('textbox')).toBeInTheDocument();
+    expect(screen.getByRole('button')).toBeInTheDocument();
+    expect(screen.getByText('Submit')).toBeInTheDocument();
+  });
+
+  // This test will work, however it won't work until async testing is introduced.
```

Comment:
Love to see that we have code written here and notes on how/what to follow up with!